### DISCUSSION TITLE
Feature: Add "Include Entities" support to Meta-Areas

### DISCUSSION THREAD

**Discussion by Bodge-IT** [jseidl/magic-areas#186] [Ideas / Feature Requests]
Sorry, didn't really know how to verbalise the feature correctly.  Hopefully this will explain:
Now that you have added a "meta location" called Global, this should also reflect HA home person presence detection.
Currently when MA detects presence in a room, both room and global states change to "detected".
When no presence detected global becomes inactive or "clear" however if I am home and HA knows it, Global should reflect that.

So by enabling 'elements to include' for global, I could add in my binary-sensor which is available to detect any persons being at home.  Alternatively, I could add trackers for individuals.  
i.e. MA may not detect anyone anywhere (if they sitting still for a while maybe) but Global should still know that someone is home if HA knows it with some certainty.

Hope that clarifies what I'm suggesting.

**Comment by cerebrate** [comment]
I also like this idea. At the current time, I'm bodging it with an interior area ("Erewhon") that doesn't actually exist containing only a template binary sensor of device-class "occupancy", which in turn reflects the state of the appropriate device-trackers, but it's kind-of an awful kluge.

**Comment by Bodge-IT** [comment]
I'm all for Bodging :-)

**Comment by jseidl** [comment]
Makes sense. That might be easy to implement, adding to the queue.
